src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  width: 1200px;
+  height: 60px;
+}
```

Comment:
there is point in checklist 
**Don't set height for header explicitly. Let the content (links) dictate it.**